sdk/appconfiguration/Azure.Data.AppConfiguration/src/Models/ConfigurationServiceSerializer.cs

```diff
@@ -80,11 +80,11 @@ public static ConfigurationSetting ReadSetting(JsonElement root)
 
             if (IsFeatureFlag(root))
             {
-                setting = new FeatureFlagConfigurationSetting();
+                setting = new FeatureFlagConfigurationSetting(root.GetRawText());
```

Comment:
Why is this needed?

It feels a little inefficient to pull the raw string out of the JsonElement and then re-parse it into a new JsonDocument a few steps later when we TryParseValue in the constructor.  Is there another way?